src/Induction/WellFounded.agda

```diff
@@ -116,6 +119,21 @@ module FixPoint
   unfold-wfRec : ∀ {x} → wfRec P f x ≡ f x λ _ → wfRec P f _
   unfold-wfRec {x} = f-ext x wfRecBuilder-wfRec
 
+------------------------------------------------------------------------
+-- Well-founded relations are asymmetric and irreflexive.
+
+module _ {_<_ : Rel A r} where
+  acc-asym : ∀ {x y} → Acc _<_ x → x < y → ¬ (y < x)
```

Comment:
For [these kind of properties](https://github.com/agda/agda-stdlib/blob/84f7366c3a038f8bcfc4ab58eaba43cb59e02d16/src/Relation/Binary/Consequences.agda#L75) the standard name would probably use `⇒` rather than `-` in the name, e.g. `acc⇒asym`, `wf⇒asym`, `wf⇒irrefl`.

Reply:
I changed the names as suggested